Yona/Yona/Others/SMSValidPasscodeLogin/Passcode/SetPasscodeViewController.swift

```diff
@@ -31,6 +38,24 @@ class SetPasscodeViewController: LoginSignupValidationMasterView {
                 
         UIApplication.sharedApplication().setStatusBarStyle(UIStatusBarStyle.LightContent, animated: false)
         
+        if isFromSettings {
+            screenNameLabel.text = NSLocalizedString("change-pin", comment: "")
+            backButton.hidden = false
+            topView.backgroundColor = UIColor.yiMangoColor()
+            self.gradientView.colors = [UIColor.yiMangoTriangleColor(), UIColor.yiMangoTriangleColor()]
+            gradientContainerView.backgroundColor = UIColor.yiMangoColor()
+            
+            let viewWidth = self.view.frame.size.width
+            let customView=UIView(frame: CGRectMake(0, 0, (viewWidth-60)/3, 2))
+            customView.backgroundColor=UIColor.yiDarkishPinkColor()
```

Comment:
This hard coded values are also used in ConfirmPasscodeViewController
So it is better to put them in a constant and use this constant instead of hardcoded raw values.
This will help when you want to change this values.